### DISCUSSION TITLE
Trying to figure a way to navigate out of RAM panel...

### DISCUSSION THREAD

**Discussion by giantclambake** [BlitterStudio/amiberry#1427] [Q&A] [answered]
As I noted, you cannot use kbd arrow keys nor game controller input, to navigate **_out_** of the RAM panel --> you can enter it ok, navigate up/down the slider widgets and change their values etc, but there is absolutely no way to exit that panel, using these input types -- mouse is required.

The only solution I can immediately think of, is including widgets as like WinUAE, that make up Advanced Memory Settings...

![ex](https://imgur.com/U5sk4TEl.png)

As my only concern here is GUI navigation, it's not immediately necessary for these Advanced Memory Settings to actually work.

What I do note however, in WinUAE, the dropdown selector for Chip RAM is always active, and linked to the 'Chip:' slider value above it in the basic settings area.

If we had that much in amiberry, it should allow for the user to be able to navigate down to the Chip RAM dropdown selector, and be able to exit this panel with kbd cursor left or dpleft ... as best I can figure it anyhow.

Would this be a solution for this navigation issue?

TIA

**Comment by midwan** [comment, marked as answer]
You can navigate _out_ of it by going to the top slider, then pushing Up once more.
It's been like that from the beginning :)

**Comment by giantclambake** [comment]
Ahh, that'd be the one direction I didn't try, as it's not intuitive to do so...(up does not mean left ;)

Thanks
